resources/report-table-formatter.js

```diff
@@ -1,13 +1,13 @@
 {
-    "Pangolin Strain (#SNPs)": function format(value) {
+    "Pangolin Strain": function format(value) {
         $(function () {
             $('[data-toggle="tooltip"]').tooltip()
         })
 
         if (value !== "no strain called") {
-            var lineage = value.split(' ')[0];
-            var link = `<a data-toggle="tooltip" data-placement="top" title="Linkout to cov-lineages" href='https://cov-lineages.org/lineages/lineage_${lineage}.html' target='_blank'>${lineage}</a>`;
-            return link + value.split(lineage).pop();
+            // var lineage = value.split(' ')[0];
```

Comment:
Is the split not needed anymore?

Reply:
pangolin doesn't output the "#SNPs" anymore. They are now using [scorpio](https://github.com/cov-lineages/scorpio) to identify constellations. A constellation is a collection of mutations which are functionally meaningful, but which may arise independently a number of times. Currently the pangolin output looks like this:

|taxon                     |lineage  |conflict|ambiguity_score   |scorpio_call          |scorpio_support|scorpio_conflict|version       |pangolin_version|pangoLEARN_version|pango_version|status   |note                                                                     |
|--------------------------|---------|--------|------------------|----------------------|---------------|----------------|--------------|----------------|------------------|-------------|---------|-------------------------------------------------------------------------|
|120586.1_NC_045512.2_RaGOO|B.1.617.2|0.0     |0.9993665933902157|Delta (B.1.617.2-like)|1.000000       |0.000000        |PLEARN-v1.2.46|3.1.8           |2021-08-09        |v1.2.46      |passed_qc|scorpio call: Alt alleles 13; Ref alleles 0; Amb alleles 0; Oth alleles 0|



Should I better remove the commented split?